tests/unit/commandlog.tcl

```diff
@@ -27,12 +27,18 @@ start_server {tags {"commandlog"} overrides {commandlog-execution-slower-than 10
         assert_equal [r commandlog len large-request] 1
 
         # for large-reply
+        set copy_avoid [lindex [r config get min-io-threads-avoid-copy-reply] 1]
+        r config set min-io-threads-avoid-copy-reply 0
```

Comment:
Reposting a comment from previous thread. Why are we disabling this here? This should work regardless of the value of the config.